data/syntax/isocpp.xml

```diff
@@ -794,7 +794,7 @@
       <itemData name="String"             defStyleNum="dsString"    spellChecking="true"  />
       <itemData name="String Char"        defStyleNum="dsSpecialChar" spellChecking="false" />
       <itemData name="Comment"            defStyleNum="dsComment"   spellChecking="true"  />
-      <itemData name="Symbol"             defStyleNum="dsNormal"    spellChecking="false" />
```

Comment:
`dsNormal` doesn't get any kind of `span`, so like `x[4]` shows up as

```html
<p><code class="sourceCode cpp">x[<span class="dv">4</span>]</code></p>
```
The `4` gets special treatment (via Decimal) but not the rest. With this change, it formats as:

```html
<p><code class="sourceCode cpp">x<span class="op">[</span><span class="dv">4</span><span class="op">]</span></code></p>
```

and now we can color punctuation.